src/actions/index.js

```diff
@@ -1,39 +1,91 @@
 import axios from "axios";
 import {
   INTERVIEW_EMAIL_SIGN_UP,
-  INTERVIEW_EMAIL_SIGN_UP_REQUEST
+  INTERVIEW_EMAIL_SIGN_UP_REQUEST,
+  INTERVIEW_EMAIL_SIGN_UP_FAILURE
 } from "./types";
 import { OTP_REQUEST, OTP_SUCCESS, OTP_FAILED } from "./types";
-import { API_URL, API_URL_OTHER } from "../config/dev";
-import { getItem } from "../helper";
+import { QUESTIONS_SUCCESS, QUESTIONS_FAILURE } from "./types";
+import {
+  CALL_HELP_REQUEST,
+  CALL_HELP_SUCCESS,
+  CALL_HELP_FAILURE
+} from "./types";
+import { API_URL } from "../config/dev";
 
 // Action for Signing Up with email to take interview test paper
 export const signUp = email => async dispatch => {
   const profile_pic = `https://pikmail.herokuapp.com/${email}?size=60`;
   const appliedEmail = email;
   dispatch({ type: INTERVIEW_EMAIL_SIGN_UP_REQUEST });
-  const res = await axios.post(`${API_URL}signup_login_fb`, {
-    email,
-    appliedEmail,
-    profile_pic
-  });
-  dispatch({ type: INTERVIEW_EMAIL_SIGN_UP, payload: res.data });
+  try {
+    const res = await axios.post(`${API_URL}signup_login_fb`, {
```

Comment:
we should not mention API_URL  everytime, you should set this as axios global default the api url